openpype/lib/execute.py

```diff
@@ -102,6 +102,10 @@ def run_subprocess(*args, **kwargs):
     if (
         platform.system().lower() == "windows"
         and "creationflags" not in kwargs
+        # shell=True already tries to hide the console window
+        # and passing these creationflags then shows the window again
+        # so we avoid it for shell=True cases
+        and not kwargs.get("shell", False)
```

Comment:
I would do explicit check for `True`
```suggestion
        and kwargs.get("shell") is not True
```